mobile-app/lib/ui/views/home/home_view.dart

```diff
@@ -11,9 +11,9 @@ class HomeView extends StatelessWidget {
   const HomeView({Key? key}) : super(key: key);
 
   static const titles = <Widget>[
-    Text('BOOKMARKED ARTICLES'),
-    Text('NEWSFEED'),
-    Text('SEARCH ARTICLES')
+    Text('Bookmarked Articles', style: TextStyle(fontSize: 15)),
```

Comment:
As this is the style of the application we would like to keep it capitalized.

Reply:
ok no problem